packages/client/tsconfig.json

```diff
@@ -1,25 +1,43 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "types": ["vite/client"],
+    "types": [
+      "vite/client"
+    ],
     "target": "ESNext",
-    "lib": ["ESNext", "DOM"],
-    "jsx": "react-jsx",
+    "lib": [
+      "ESNext",
+      "DOM",
+      "DOM.Iterable"
+    ],
+    "module": "ESNext",
     "esModuleInterop": true,
-    "allowSyntheticDefaultImports": true,
-    "module": "esnext",
     "strict": true,
-    "baseUrl": ".",
```

Comment:
The config option [`baseUrl`](https://www.typescriptlang.org/tsconfig/#baseUrl) is no longer required to work paths as of TS 4.1., and it was designed to be used with AMD module loaders in the browser (however ESM has taken over here 🤷 ) requirejs and AMD has been [dying](https://www.reddit.com/r/javascript/comments/46sbd2/is_amd_requirejs_dying/) a slow death for some years now

![Screenshot 2024-09-09 at 01 28 04](https://github.com/user-attachments/assets/df6c0d00-debe-49d0-b33b-6e6d57a64296)